python-lib/nlp_utils.py

```diff
@@ -3,16 +3,37 @@
 import unicodedata
 
 
-def get_keyword(text: AnyStr, normalize_case: bool) -> AnyStr:
+def get_text_case(text: AnyStr, normalize_case: bool) -> AnyStr:
     """Return text in its wanted-case form"""
     return text.lower() if normalize_case else text
 
+def get_attribute(lemmatize: bool) -> AnyStr:
```

Comment:
Same, the function name does not convey what the function does. Here it's "get_token_text_or_lemma"

Reply:
I don't really agree on this one, "get_token_text_or_lemma" says you'll have the token text or the token lemma, but here you'll have the string  attribute "text" or "lemma". Perhaps get_token_attribute?  0765d3c